src/pages/defineTimeRestrictions.tsx

```diff
@@ -21,7 +21,7 @@ export const getFieldsets = (errors: ErrorInfo[]): RadioConditionalInputFieldset
     const timeRestrictionsFieldset: RadioConditionalInputFieldset = {
         heading: {
             id: 'define-time-restrictions',
-            content: 'Is there a start time or an end time to this ticket?',
+            content: 'Is there a start and end time to this ticket?',
```

Comment:
Are we sure about this? If the user only has a start time, and we want them to click yes, this question's correct answer for that would be no, as its asking if they have both?

Reply:
Or is this one where both are required.